openlibrary/catalog/marc/tests/test_data/bin_expect/710_org_name_in_direct_order.json

```
@@ -0,0 +1,41 @@
+{
+  "publish_date": "2004",
+  "publish_country": "cc",
+  "languages": [
+    "chi"
+  ],
+  "title": "\u4e2d\u56fd\u8bd7\u6b4c\u7814\u7a76\u52a8\u6001",
```

```suggestion
  "title": "中国诗歌研究动态",
```
This would be easier to read as UTF-8 than escaped Unicode code points